content/posts/networkx/outreachy2023/contribution-phase/index.md

```diff
@@ -34,10 +34,12 @@ If this is your first time contributing to an open-source project you may feel o
 - _Learn about the project:_ Understanding the project is a process that may take some time. Don’t rush it! You don’t need to understand the entire codebase in a day. The most important things that you need to know only will take you a few hours to go through: Learn about the project mission and values,
   community rules, and contribution process. In NetworkX all you need to know is here: <https://networkx.org/documentation/stable/developer/index.html>
 
-- _Start contributing right away:_ You don't need to understand every part of the project to make valuable contributions. Start small and use that experience to level up your contributions. At the beginning of the contribution phase, some good first issues are added. Work on them first and then start opening your own issues (Don’t forget to link your PR with the issues so they can be automagically closed). Also, record your contributions on the Outreachy website as you submit them. I only recorded all contributions at the end and that took me a lot of time.
+- _Start contributing right away:_ You don't need to understand every part of the project to make valuable contributions. Start small and use that experience to level up your contributions. At the beginning of the contribution phase, some good first issues are added. Work on them first and then start opening your own issues (Don’t forget to link your PR with the issues so they can be automagically closed). Also, record your contributions on the Outreachy website as you submit them. I only recorded all contributions at the end and that took me a lot of time. If you struggle to find issues or ideas for contributions here are my contributions at NetworkX: <https://github.com/networkx/networkx/pulls?q=is%3Apr+paulitapb>
```

Comment:
Thank you for adding that. Just a note, no need to change anything per se. Contributions are not only PRs 😃 Discussions you could have had on issues are sometimes even more important than PRs themselves. All sorts of interactions are "contributions" as they keep the project moving.